packages/hardhat-plugin/test/use-module.ts

```diff
@@ -33,10 +32,6 @@ describe("useModule", function () {
           args: [],
         });
 
-        if (!isCallable(foo)) {
```

Comment:
Our changes to the useModule return types removes the need for this check.

Reply:
Zoey's enhancements to support typescript included constraining the allowed returnable futures. An attempt to return a non-contract future from the module will error. This case is covered by a core test.

The upshot of these changes is we no longer need this if check to satisfy typescript, and the error case would never be hit because useModule would have thrown earlier.